Tasks/KubernetesManifestV0/src/utils/KubernetesObjectUtility.ts

```diff
@@ -287,6 +287,45 @@ function setImagePullSecrets(inputObject: any, newImagePullSecrets: any) {
     return;
 }
 
+export function updateImageDetails(inputObject: any, containers: string[]) {
+    if (!inputObject || !inputObject.spec || !containers) {
+        return;
+    }
+
+    if (!!inputObject.spec.template && !!inputObject.spec.template.spec) {
+        if (!!inputObject.spec.template.spec.containers) {
+            updateContainers(inputObject.spec.template.spec.containers, containers);
+        }
+        if (!!inputObject.spec.template.spec.initContainers) {
+            updateContainers(inputObject.spec.template.spec.initContainers, containers);
```

Comment:
Do we need to update initContainers? init container are different then your deployment containers.

Reply:
If we remove this it could be a breaking change, because it was replacing earlier.